conf/vmware.yaml.template

```diff
@@ -1,7 +1,14 @@
 VMWARE:
   # VMware to be added as a compute resource
   # VCENTER: vmware vcenter URL
-  VCENTER:
+  VCENTER7:
+    HOSTNAME:
+    HYPERVISOR:
+    MAC_ADDRESS:
```

Comment:
Could you add a comment here for mac addr field to know which vm it belongs or vm_name filed in this template?

Reply:
Updated